app/controllers/users_controller.rb

```diff
@@ -21,6 +22,17 @@ def edit
     @trophies = @user.trophy_files
   end
 
+  def destroy
```

Comment:
@pbinkley you had asked about how to handle user sessions.  I think this is what you need: ```sign_out :user``` https://github.com/plataformatec/devise/blob/master/lib/devise/controllers/sign_in_out.rb#L47-L67